assets/css/style.css

```diff
@@ -37,99 +39,99 @@
     transform: translate(-50%, -50%);
     transition: transform .6s, opacity .8s, top .6s;
 }
-.ham-bar:hover{
+
+.ham-bar:hover {
    background-color: white; 
 }
 
-.bar-top{
+.bar-top {
     top: 25%;
     background-color: whitesmoke;
 }
 
-.bar-bottom{
+.bar-bottom {
     top: 75%;
     background-color: whitesmoke;
 }
 
-#hamburger-menu.active .bar-top{
+#hamburger-menu.active .bar-top {
     transform: translate(-50%, -50%) rotate(-315deg);
     top: 50%;
 }
 
-#hamburger-menu.active .bar-mid{
+#hamburger-menu.active .bar-mid {
     opacity: 0;
 }
 
-#hamburger-menu.active .bar-bottom{
+#hamburger-menu.active .bar-bottom {
     transform: translate(-50%, -50%) rotate(-225deg);
     top: 50%;
 }
 
 /* off Screen */
-.off-screen{
+.off-screen {
     position: fixed;
     bottom: -100%;
     height: 100%;
     width: 100%;
     left: 0px;
     z-index: 99;
-    background-color: #22b4abdb;
+    background-color: #22b4ab;
```

Comment:
This is a good candidate for a var variable. Please change it into var in the next branch.